plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/eap/OpenShiftEapModulesController.java

```diff
@@ -50,14 +52,14 @@ private void deleteMarkers(String suffix) throws CoreException {
 		final File localDeploymentDirectory = new File(getDeploymentOptions().getDeploymentsRootFolder(true));
 		Stream.of(localDeploymentDirectory.listFiles())
 			.filter(p->p.getName().endsWith(suffix))
-			.forEach(p->p.delete());
+			.forEach(File::delete);
```

Comment:
in what is change this superior? To me this is just an alternative way to achieve the same without added benefit.

Reply:
SonarLint is configured to push users to use the method reference writing.
I don't have a strong opinion on that. I find it a little bit more readable but can understand that it can be a matter of taste.
I will revert it

Reply:
I dont have an opinion either, I was just wondering. Works for me either way.